### PR TITLE
fix: (Core) Remove redundant tabbable item on multi input, remove default width limit, improve ShowAll button

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/4875
fixes: https://github.com/SAP/fundamental-ngx/issues/4873
#### Please provide a brief summary of this pull request.
There is removed unnecessary tabindex on `tokenizer` component.
Way how the focus indicator is added has been changed.
API:
- added `bodyMaxWidth` Input to MultiInput
- added `tokenizerFocusable` Input to Tokenizer
- showAll Button is improved - keydown support + visual fix
- Breaking Change - there is removed input named `tokenizerHasFocus`. It's not needed to be `@Input`

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by droshev — APPROVED** [review]

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 1]
```diff
@@ -432,10 +436,18 @@ export class MultiInputComponent implements
...
-    showAllClicked(event: MouseEvent): void {
+    showAllClicked(event: Event): void {
         event.preventDefault();
         event.stopPropagation();
         this._applySearchTermChange('');
+        this.searchInputElement.nativeElement.focus();
+    }
+
+    /** @hidden */
+    showAllKeyDown(event: KeyboardEvent): void {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/multi-input/multi-input.component.ts:447]: shouldn't these two be prefixed with `_` as they are hidden?

File: libs/core/src/lib/token/tokenizer.component.html [hunk 2]
```diff
@@ -1,14 +1,11 @@
 <div
     class="fd-tokenizer fd-tokenizer--scrollable"
-    tabindex="0"
-    (focus)="tokenizerHasFocus = true"
-    (blur)="tokenizerHasFocus = false"
-    [ngClass]="{ 'is-focus': tokenizerHasFocus, 'fd-tokenizer--compact': compact }"
+    [ngClass]="{ 'is-focus': _tokenizerHasFocus, 'fd-tokenizer--compact': compact }"
```
Comment (InnaAtanasova) [inline libs/core/src/lib/token/tokenizer.component.html:3]: I think we decided to follow the format:
[class.is-focus] = "_tokenizerHasFocus"
[class.fd-tokenizer--compact] = "compact"

**Comment by JKMarkowski** [comment]
@InnaAtanasova Thank You for review, could you check that again? :)

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**